### PR TITLE
moved mute API to SDK sharedInstance

### PR DESCRIPTION
**Pull request by 2energycell**
changes in adapter following SDK changes;
this change is mandatory;

### CODE REVIEW

**Comment by 2energycell** [comment]
@TomerFyber - FYI, please review;
@joshuajplagonera please review and approve;

File: adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m [hunk 1]
```diff
@@ -77,7 +77,7 @@ GADVersionNumber GADMAdapterFyberVersionFromString(NSString *_Nonnull versionStr
     builder.spotID = spotID;
     builder.timeout = 10;
     builder.userData = extras.userData;
-    builder.muteAudio = extras.muteAudio;
+    IASDKCore.sharedInstance.muteAudio = extras.muteAudio;
```
Comment (joshh-devrel) [inline adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m:80]: @2energycell @TomerFyber 
Can you confirm if this is the only place we need to update to use `IASDKCore.sharedInstance`?

**Review by TomerFyber — APPROVED** [review]

File: adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m (hunk 1 above)
Comment (joshh-devrel) [reply adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m:80]: @2energycell @TomerFyber 
Gentle ping on the above comment.

Also, we can move this line before building the request (i.e. line 75).
We should probably check if the `muteAudio` is set before setting it in the `sharedInstance`:
```
  if (extras.muteAudio) {
    IASDKCore.sharedInstance.muteAudio = extras.muteAudio;
  }
 ```

Comment (joshh-devrel) [reply adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m:80]: @2energycell @TomerFyber Any update on the above comments?

Comment (joshh-devrel) [reply adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m:75]: @2energycell @TomerFyber Gentle bump on the above comments.

**Comment by joshh-devrel** [comment]
@2energycell @TomerFyber Can you take a look at the existing comment? Thanks!

**Comment by 2energycell** [comment]
closing this one, please merge this PR instead: https://github.com/googleads/googleads-mobile-ios-mediation/pull/409

**Pull request closed without merging**